app/routes/projects-list.js

```diff
@@ -1,6 +1,14 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  currentUser: Ember.inject.service(),
+
+  beforeModel() {
+    this._super(...arguments);
+    let user = this.get('currentUser.user');
+    user.get('skill');
+  },
```

Comment:
I thought that the user's skills were not loading explicitly from the user skills service.